### PR TITLE
Fixes Space UI changes 

### PR DESCRIPTION
**Pull request by RutvikJ77**
## What is the change?
Updated the spacing arrangements on the web page.

## Related issue?
issue: #159 

## How was it tested?
Updated directly through the Developer tools. Unfortunately, there has been an issue running the project locally showing connection refused to the DB.

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [x] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video:
Screenshot 2021-02-17 at 6 09 56 PM
Screenshot 2021-02-17 at 6 10 05 PM

### CODE REVIEW

**Comment by RutvikJ77** [comment]
Sorry was not able to run the app locally which hindered the dev. process. 😓

File: views/partials/footer.ejs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 </div>
 <br><br>
-<footer class="text-center text-white container" style="background-color:#07527e;width:100%;bottom: 0; margin-top: auto;">
+<footer class="text-center text-white container" style="background-color:#07527e;width:100%;bottom: 0; margin-top: auto;padding:10px;">
```
Comment (Pranav016) [inline views/partials/footer.ejs:4]: Don't hard code values, that affects responsiveness, using relative units of measurements like `vh` or `vw` or `rem`.

File: views/partials/footer.ejs [hunk 2]
```diff
@@ -15,7 +15,7 @@
       <h5 class="text-uppercase"><b style="color:rgb(255, 231, 10);">Quick Links</b></h5>
       <ul class="list-unstyled mb-0" >
         <li>
-          <a href="#!" style="color:white;"><i class="fa fa-award"></i>Careers</a>
+          <a href="#!" style="color:white;"><i class="fa fa-award" style="padding-right: 3px;"></i>Careers</a>
```
Comment (Pranav016) [inline views/partials/footer.ejs:18]: Use relative units of measurements.

**Review by Pranav016 — CHANGES_REQUESTED** [review]
* How did you make UI changes without even running the app, these changes look redundant. Show me what error you are facing to run the app, I'll help you 👍 
* There is a new pull request for making the website responsive, after that gets merged it would be even better to improve the UI then.
* In the screenshot you shared, the footer looks like it is stuck at the top. I don't get it..
* The commit messages are also wrong. What is meaning of `Chore:` ?
* Change base branch of the PR to develop, don't make the PR to master. Check contribution guidelines.

**Comment by Pranav016** [comment]
I am closing this PR for now, reopen if required.

**Comment by RutvikJ77** [comment]
- Okay! I will attach the error.
- Okay! I will try to improve the PR once the responsive one gets merged.
- They are both different screenshots, mistakenly I uploaded the footer one first and then the spacing one.
- I am using the udacity commit style guide. Chore means a task.
- My bad, I will make sure I check the merging branch as well. 
- I agree, I will use the responsive values.

**Comment by Pranav016** [comment]
> * Okay! I will try to improve the PR once the responsive one gets merged.

You can work on this now, Idk when the responsive one gets merged. Since I am clearing assignees on issues more than 4-5 days old so let me know if you're still working..

**Pull request closed without merging**